scenario/utils/index.ts

```diff
@@ -257,6 +258,10 @@ export async function isBulkerSupported(ctx: CometContext): Promise<boolean> {
   return bulker == null ? false : true;
 }
 
+export function isInNetwork(ctx: CometContext, networks: string[]): boolean {
```

Comment:
we should replace this with the more powerful `matchesDeployment`fn from #587: 

https://github.com/compound-finance/comet/pull/587/files#diff-33630834798399f8c06541165f332a3c60939d28f72c8a664a4e4e116737d02aR265

Reply:
Really wish we could get rid of `matchesDeployment` though and find a way to do things through constraints

Reply:
most often, we need `matchesDeployment`because there's a scenario that assumes a particular relationship between the base asset and a collateral asset.

for example, if you have a scenarios that 1) supplies 1,000 of `$asset0` and then 2) borrows 100 of the base asset.

that works for many of our deployments, because the base asset is a stablecoin and the first asset is worth more than $1/unit.

but you need way more collateral than that when the base asset is WETH.

Reply:
we could add a helper function that takes in a comet instance and a number of base units and it returns you the amount of the first collateral asset that is needed in order to cover a borrow of that many base units

Reply:
Then we're getting away from using real numbers in scens... I'm starting to think a PriceConstraint, though relatively complicated, might solve a bunch of these problems more elegantly

Reply:
#615